pkg/cmd/openshift-kube-apiserver/configdefault/kubecontrolplane_default.go

```diff
@@ -5,10 +5,50 @@ import (
 	"os"
 	"path"
 
+	"github.com/golang/glog"
 	kubecontrolplanev1 "github.com/openshift/api/kubecontrolplane/v1"
 	"github.com/openshift/library-go/pkg/config/configdefaults"
 )
 
+func ResolveDirectoriesForSATokenVerification(config *kubecontrolplanev1.KubeAPIServerConfig) {
+	// kube doesn't honor directories, but we want to allow them in our sa token validators
+	resolvedSATokenValidationCerts := []string{}
+	for _, filename := range config.ServiceAccountPublicKeyFiles {
+		file, err := os.Open(filename)
+		if err != nil {
+			resolvedSATokenValidationCerts = append(resolvedSATokenValidationCerts, filename)
```

Comment:
when can this happen? Why do we ignore the error?

Reply:
adding godoc to explain.